ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionValidator.java

```diff
@@ -291,6 +299,18 @@ public ValidationResult<TransactionInvalidReason> validateTransactionsBlobs(
           "transaction blob transactions cannot have a to address");
     }
 
+    if (transaction.getVersionedHashes().isEmpty()) {
+      return ValidationResult.invalid(
+          TransactionInvalidReason.INVALID_BLOBS,
+          "transaction blob transactions must specify one or more versioned hashes");
+    }
+
+    return ValidationResult.valid();
+  }
+
+  public ValidationResult<TransactionInvalidReason> validateTransactionsBlobs(
+      final Transaction transaction) {
+
```

Comment:
@jflo can you make sure this section won't cause problems?  I needed to validate a type 3 / blob transaction has a versioned hash but it didn't have any of the blob data to go with it.  Seems like a typical block replay use case.